website/settings/defaults.py

```diff
@@ -594,6 +595,11 @@ class CeleryConfig:
                 'schedule': crontab(minute=0, hour=6),  # Daily 1:00 a.m.
                 'kwargs': {'yesterday': True}
             },
+            'send_tou_notif': {
```

Comment:
I'm not sure we want this to be scheduled by `beat`. Currently, `PeriodTask` objects created by this schedule need to be disabled manually, not just removed from the settings.